MDSassignment/guestbook/post/views.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render, redirect
+from django.contrib import messages
+from .models import Post 
+from .forms import PostForm
+import time
+from django.utils import timezone
+
+# Create your views here.
+def post_page(request):
+	if request.method == 'POST':
+		form = PostForm(request.POST)
+		if form.is_valid():
+			posts = Post()
```

Comment:
Since `PostForm` is a `ModelForm` you can call `save()` on the form and it will return the post instance. No need to create and save the Post instance yourself:

```python
if form.is_valid():
    post = form.save()
```